python/lsst/ts/imsim/imsimCmpt.py

```diff
@@ -518,6 +518,8 @@ def _mapOpdToZk(self, rotOpdInDeg, numOpd):
             else:
                 opdRot = opd
 
+            # opdRot = np.fliplr(opdRot)
```

Comment:
Is this commented out because of the Zernike flips done earlier?

Reply:
Yes, I'll remove this.